src/lib/Scenes/Artwork/Components/__tests__/PartnerCard-tests.tsx

```diff
@@ -41,6 +41,21 @@ describe("PartnerCard", () => {
     expect(component.find(Image)).toHaveLength(1)
   })
 
+  it("renders partner type", () => {
+    const component = mount(
+      <Theme>
+        <PartnerCard relay={{ environment: {} } as RelayProp} artwork={PartnerCardArtwork} />
+      </Theme>
+    )
+
+    expect(
+      component
+        .find(Sans)
+        .at(0)
+        .text()
+    ).toMatchInlineSnapshot(`"At gallery"`)
+  })
```

Comment:
Should we do a test for the other partner types too?

Reply:
Sure